deploy/eck-stack/charts/eck-agent/templates/elastic-agent.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     {{- include "elasticagent.labels" $ | nindent 4 }}
   annotations:
-    eck.k8s.elastic.co/license: enterprise
+    eck.k8s.elastic.co/license: basic
```

Comment:
We could also remove the annotation completely, but I thought it was nice to keep it explicit here. Happy to change if you think it's useless.